03LessonThree/in-class-project/daily-planet.html

```diff
@@ -5,6 +5,41 @@
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
+      <header>
+        <h1>Daily Planet</h1>
+        <nav>
+          <a href="http://espn.go.com/">Sports</a>
+          <a href="http://kxan.com/">Local</a>
+          <a href="http://www.wsj.com/">Economy</a>
+        </nav>
+      </header>
+      <main>
+        <section>
+          <article>
+            <video width="320" height="240" autoplay loop>
+              <source src="https://i.imgur.com/wFovN4O.mp4" type="video/mp4">
+            </video>
+          </article>
+          <article>
+            <h2>Bill Clinton Loves Balloons!</h2>
+            <p>Last night at the Democratic National Convention, we learned a number of things. Chelsea Clinton kind of sounds like Mia Thermopolis from The Princess Diaries if you close your eyes and squint your ears. "Believing in science" is somehow not a given for a presidential candidate in 2016. And most importantly, Bill Clinton, possible future First Gentleman of the United States, loves balloons.</p>
+            <br><br>
+            <p>Bill Clinton loves balloons truly, madly, deeply. He loves them like they are the morning sun and the evening moon. He loves them the way most people love finding a $50 bill in their winter coat. The internet, a collection of humans with the skill to purify crude wells of joy into barrels of pure energizing happiness, did excellent work with this revelation.</p>
+          </article>
+        </section>
+        <aside>
+          <img src="http://static4.businessinsider.com/image/56f1953652bcd028008b8102-1000-1415/tollthis.jpg" alt="Burger King">
+        </aside>
+      </main>
+      <footer>
+        <a href="mailto:jamesjaypuckett@gmail.com">Email</a>
+        <br>
+        <p>+1(555)555-5555</p>
+        <div class="footright">
+          <a href="https://www.facebook.com/james.puckett.35"><img src="https://cdn0.iconfinder.com/data/icons/yooicons_set01_socialbookmarks/256/social_facebook_box_blue.png" alt="Facebook"></a>
+          <a href="#"><img src="http://files.softicons.com/download/social-media-icons/clean-simple-social-icons-by-creative-nerds/png/256x256/Twitter.png" alt="Twitter"></a>
+          <a href="#"><img src="http://www.democrathotsprings.com/Instagram-icon-small.png" alt="Instagram"></a>
+        </div>
+      </footer>
     </body>
-</html>
\ No newline at end of file
+</html>

```

Comment:
The only project requirement you are missing is the "Add Copyright and Year to footer". If you can add and recommit, I can bump your grade up to 100.